record_store.gemspec

```diff
@@ -31,11 +31,14 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'dnsimple', '~> 4.4.0'
   spec.add_runtime_dependency 'google-cloud-dns'
   spec.add_runtime_dependency 'ruby-limiter', '~> 1.0', '>= 1.0.1'
+  spec.add_runtime_dependency 'ns1'
+
 
   spec.add_development_dependency 'rake'
   spec.add_development_dependency 'bundler'
   spec.add_development_dependency 'mocha'
   spec.add_development_dependency 'vcr'
   spec.add_development_dependency 'pry'
+  spec.add_development_dependency 'byebug'
```

Comment:
Is this really required ?

Reply:
Not necessarily, I think it shouldn't be there. Will remove that line!

Reply:
Consider keeping it to pay it forward to future maintainers who need to pop into a debugger.  This is probably why `pry` was already present.  To @XaF 's point, though, we should never check in code that *`require`s* `byebug`.

Reply:
I did not realize it was part of `add_development_dependency` as the diff did not show the lines above in the code.
You can keep this around and ignore my comment, I'll resolve.